src/components/content/policies/Policies.tsx

```diff
@@ -0,0 +1,281 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ */
+
+import React, { useEffect, useState } from 'react';
+import { Button, Image, Modal, Popconfirm, Popover, Space, Table } from 'antd';
+import { ColumnsType } from 'antd/es/table';
+import '../../../styles/policies.css';
+import useListPoliciesManagementServiceQuery from './useListPoliciesManagementServiceQuery';
+import { AbstractCredentialInfo, CloudServiceProvider, PolicyVo } from '../../../xpanse-api/generated';
+import { ColumnFilterItem } from 'antd/es/table/interface';
+import { PoliciesManagementServiceListSkeleton } from './PoliciesManagementServiceListSkeleton';
+import PoliciesManagementServiceListError from './PoliciesManagementServiceListError';
+import {
+    AppstoreAddOutlined,
+    CloseCircleOutlined,
+    EditOutlined,
+    InfoCircleOutlined,
+    SyncOutlined,
+} from '@ant-design/icons';
+import { EditPolicy } from './EditPolicy';
+import { PolicyDetails } from './PolicyDetails';
+import { updateCreateOnFilters, updateCspFilters } from './policiesParams';
+import PolicyDeletePolling from './delete/PolicyDeletePolling';
+import { useDeletePolicyRequest } from './delete/useDeletePolicyRequest';
+import { cspMap } from '../order/types/CspLogo';
+
+function Policies(): React.JSX.Element {
+    const [id, setId] = useState<string>('');
+    const [currentPolicyService, setCurrentPolicyService] = useState<PolicyVo | undefined>(undefined);
+    const [cspFilters, setCspFilters] = useState<ColumnFilterItem[]>([]);
+    const [createOnFilters, setCreateOnFilters] = useState<ColumnFilterItem[]>([]);
+    const [isDeleting, setIsDeleting] = useState<boolean>(false);
```

Comment:
please try to remove both isDeleting and isDeletingCompleted states. We can directly use deletePoliciesManagementServiceRequest states for these checks.